core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -3797,10 +3797,16 @@ class KafkaApis(val requestChannel: RequestChannel,
       )
   }
 
+  private def isConsumerGroupProtocolEnabled(): Boolean = {
+    val featureLevel = metadataCache.features().finalizedFeatures().getOrDefault(GroupVersion.FEATURE_NAME, GroupVersion.GV_1.featureLevel)
```

Comment:
is the default version 1 since we always have some group coordinator?

Reply:
Right. I thought that we could use version 1 as the default version for all the existing APIs (old protocol, etc). If not provided, we can still default to it.

I think that in the KIP, we said that we wanted to use version 0 for this but as 0 also means disabled I found it weird. Let me know what you think.

Reply:
I think this is reasonable. One of us should probably update the KIP so there aren't more questions on this 😅

Reply:
We could also use 0 as the default and 1 for the new protocol. It would also work for me.

Reply:
I really don't have a strong opinion either way 😄